float32 distanceFromBotLeftToHomeLeft
float32 distanceFromBotLeftToHomeRight
float32 distanceFromBotRightToHomeLeft
float32 distanceFromBotRightToHomeRight
float32 maxDistanceToHome
float32 minDistanceToHome
float32 distanceToHome
float32 angleToHome
float32 orientation
float32 distanceToStagingPoint
float32 angleToStagingPoint

